src/main/kotlin/blackjack/domain/gamer/Gamers.kt

```diff
@@ -29,6 +37,13 @@ class Gamers private constructor(
     companion object {
         private const val PLAYER_INDEX = 1
 
+        fun init(playerNames: List<String>): Gamers {
+            val dealer = Dealer.from(Cards())
+            val players = playerNames.map { Player.of(it.trim(), Cards()) }
+            val gamers = listOf(dealer).plus(players)
+            return Gamers(gamers)
+        }
+
         fun from(gamers: List<Gamer>): Gamers {
             return Gamers(gamers)
         }
```

Comment:
```suggestion
        fun from(dealer, List<Player>): Gamers {
            return Gamers(gamers)
        }
```
팩토리 자체에서도 제약을 줄수 있을거같아요.